### ISSUE TITLE
Center Loss in TF 2.0

### ISSUE THREAD

**Issue by suraj-maniyar** [tensorflow/tensorflow#37410]
I am trying to implement center loss in Tensorflow 2.0 by referring to code in following repositories:

- https://github.com/handongfeng/MNIST-center-loss

- https://github.com/mjDelta/keras-center-loss-MNIST

- https://github.com/Kakoedlinnoeslovo/center_loss

I noticed that some arguments in the method "self.add_update((self.centers,new_centers),x)" when creating a custom layer, are deprecated. The code (linked above) does not throw any error, but the centers are never updated when run in Tensorflow 2.0. 

You can find the related description for the problem here:
https://github.com/tensorflow/tensorflow/blob/v2.1.0/tensorflow/python/keras/engine/base_layer.py#L1170-L1259

I am trying to update the weights of the layer after every forward pass so I can use it as tf.Variable which can persist the updated centroids for the next iteration.

Any help on this would be appreciated.
Thanks

**Comment by ravikyram** [comment]
@suraj-maniyar 

Thanks for reporting the issue. Can you please share the simple standalone code to reproduce the issue. It helps us in localizing the issue faster. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by ravikyram** [comment]
@suraj-maniyar 

Any update on this issue please. Thanks!

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by pgaggar** [comment]
Is there a solution to this? I'm also facing exactly the same problem.

**Comment by pgaggar** [comment]
To update this, we need to use 

> tf.keras.backend.update(x, new_x)

 as the argument to the add_update function, as it expects an update operation.
